### PR TITLE
ACL CLI for token/policy create

### PR DESCRIPTION
**Pull request by dadgar**

### CODE REVIEW

File: command/acl_token_create_test.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	defer srv.Shutdown()
+
+	// Bootstrap an initial ACL token
+	token := srv.Token
+	assert.NotNil(token, "failed to bootstrap ACL token")
+
+	ui := new(cli.MockUi)
+	cmd := &ACLTokenCreateCommand{Meta: Meta{Ui: ui, flagAddress: url}}
+
+	// Request Job List without providing a valid token
```
Comment (chelseakomlo) [inline command/acl_token_create_test.go]: I think this is the incorrect comment- same with below

File: command/acl_policy_apply_test.go [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+	// Get a file
+	f, err := ioutil.TempFile("", "nomad-test")
+	assert.Nil(err)
+	defer os.Remove(f.Name())
+
+	// Write the policy to the file
+	err = ioutil.WriteFile(f.Name(), []byte(policy.Rules), 0700)
+	assert.Nil(err)
+
+	// Request Job List without providing a valid token
```
Comment (chelseakomlo) [inline command/acl_policy_apply_test.go]: I think this the incorrect comment

File: command/acl_token_create.go [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+	}
+
+	// Get the HTTP client
+	client, err := c.Meta.Client()
+	if err != nil {
+		c.Ui.Error(fmt.Sprintf("Error initializing client: %s", err))
+		return 1
+	}
+
+	// Get the bootstrap token
```
Comment (dadgar) [inline command/acl_token_create.go]: // Create the token

File: commands.go [hunk 4]
```diff
@@ -36,6 +36,26 @@ func Commands(metaPtr *command.Meta) map[string]cli.CommandFactory {
 				Meta: meta,
 			}, nil
 		},
+		"acl policy": func() (cli.Command, error) {
+			return &command.ACLPolicyCommand{
+				Meta: meta,
+			}, nil
+		},
+		"acl policy apply": func() (cli.Command, error) {
```
Comment (dadgar) [inline commands.go]: The subcommands should be filtered in main.go

File: main.go [hunk 5]
```diff
@@ -44,7 +44,7 @@ func RunCustom(args []string, commands map[string]cli.CommandFactory) int {
 	// users should not be running should be placed here, versus hiding
 	// subcommands from the main help, which should be filtered out of the
 	// commands above.
-	hidden := []string{"check", "executor", "syslog"}
+	hidden := []string{"check", "executor", "syslog", "apply"}
```
Comment (dadgar) [inline main.go]: Put in in the above commandsInclude list.

`acl policy, acl policy apply, acl token, acl token create`

File: command/acl_policy.go [hunk 6]
```diff
@@ -0,0 +1,19 @@
+package command
```
Comment (dadgar) [inline command/acl_policy.go]: Actually missing one command `nomad acl`. Look at the nomad operator command.

**Pull request merged**